src/components/list/MenuList.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import MenuItem from '../list/MenuItem';
+
+const SIZE = {
+  SMALL: 'small',
+  NORMAL: 'normal',
+  LARGE: 'large'
+};
+
+const MenuList = ({items, size}) => {
+  const listClass = classNames('sg-menu-list', {
+    [`sg-menu-list--${size}`]: size !== SIZE.NORMAL
```

Comment:
nothing breaks, the list has a default size (there are only additional classes for small and large size)